app/controllers/home_controller.rb

```diff
@@ -1,41 +1,31 @@
 class HomeController < ApplicationController
   def index
-    # Get the top 10 posts on the front page in the format of {title: upvotes}
-    @subscriber = Subscriber.hashify(Subscriber.top_ten)
+    @subscribers = Subscriber.hashify(Subscriber.top_ten)
     # Timeframes allowed in current dropdown list
-    @timeframe = { '5 minutes' => 5,
+    @timeframe = { '5 minutes'  => 5,
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.